docs/examples/bar-line-chart.html

```diff
@@ -9,6 +9,16 @@
       rel="preconnect"
       crossorigin="anonymous"
     />
+    <link
```

Comment:
In most files, you add `orange-helvetica.min.css`. However, if `boosted.css` is not added, I think it won't be applied (not tested).

Reply:
Let's keep your version as it's not that important that we load the right fonts when the users (in the documentation) switch to v4. However, it should be handled in https://github.com/Orange-OpenSource/ods-charts/pull/173 by switching between `orange-helvetica.min.css` and `orangeHelvetica.min.css` depending on the Boosted version.
I'm resolving this conversation.